src/app/content/highlights/components/cardStyles.ts

```diff
@@ -147,7 +147,7 @@ export const mainCardStyles = css`
     `;
   }}
 
-  /* for some reason isTocOpen prop will not refresh after first page load and is null */
+  /* isTocOpen means toc is open on the desktop and for mobile, there is a different media query */
```

Comment:
isTocOpen is used by both desktop and mobile. for highlight cards, within these highlighting media queries, we know we're on a desktop screen, so we know that `isTocOpen === null` means the sidebar is shown. thats why the condition is `props.isTocOpen === null || props.isTocOpen`.